### DISCUSSION TITLE
Duda UI diseño

### DISCUSSION THREAD

**Discussion by RodolfoMorquecho** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#195] [Q&A] [answered]
Hola, mi duda es demasiado básica y agradecería alguien me solventara lo siguiente.
En la imagen  pueden ver que hay un mockup que hice de la parte de UX obvio sabemos que la prioridad es hacer una representación funcional o maquetado mas que estética de nuestro sitio web y como se observa el texto esta simbolizado con lineas grises, pero ahora que estoy a mediados de la parte de UI es valido también representar el texto de esa forma o ahí debo ser necesariamente muy explicito y poner texto real para secciones no tan relevantes?

![Ejemplo UX](https://user-images.githubusercontent.com/99112892/156282364-7522b89b-a783-4d19-9876-1a3081fe412d.png)

**Comment by JuanSalvador73** [comment, marked as answer]
Si estás trabajando solo UX, puedes dejarlo así como texto simulado. Si estás trabajando UI, si es recomendable ya tener el texto "final" ya que este diseño de UI te indica como va a quedar la apariencia de tu App junto con la gama de colores a usar.

No creo que afecte mucho que lo entregues como etapa final de UI en LaunchX FrontEnd, solo recuerda que entre mas detalles de tu trabajo muestres mas claro vas a ejemplificar como va a funcionar tu App, y supongo que eso también influirá en la oferta de vacantes que han prometido. 

Por cierto, no hay "secciones irrelevantes" en un maquetado de App de UI, ya que todo paso es importante sobretodo al hacer la programación final en BackEnd. Lo que puedes hacer es solo mostrar las pantallas que consideres mas importantes para mostrar la vista y "vender" la idea, eso si sabiendo que ya tienes todo el flujo perfectamente organizado en tu diagrama con la lógica de cada paso definida.

Suerte :D

**Reply by RodolfoMorquecho** [reply]
Antes que nada gracias por responder Juan, lo que me paso es que ya estaba haciendo todo con texto real en la UI pero me entro la duda si debía ser igual simulado como en la UX pero tiene lógica lo que me dices solo que soy novato como muchos por acá en estos temas.
En cuanto a las secciones irrelevantes tal ves utilice un termino despectivo para darme a entender pero concuerdo que no existe ese termino cuando quieres vender una idea y es mejor enfocarte en mostrar lo necesario. 
Saludos!

**Reply by JuanSalvador73** [reply]
No te apures Rodolfo, en cierta forma todos los Explorers aquí somos novatos, incluso aquellos con mas experiencia están aprendiendo algo, finalmente se trata de eso.

Y no, jamás consideré siquiera que te referías con términos peyorativos cuando decías que el término sonaba despectivo. Me refería a que sobretodo en la parte de programación en sitios y en aps, los detalles minúsculos cuentan mucho, cosas tan simples como una pantalla de "error", que te faltó añadir un botón de salida cuando terminaste o como le indicas al usuario que recibirá una notificación son importantes, creéme, lo aprendí a la mala hace mucho. 

Cualquier cosa aquí andamos todos los Explorers. Suerte, seguimos juntos. :D

**Comment by doguedogue** [comment]
Se ve muy bien, como dices una cosa es el wireframe (maqueta || bosquejo || borrador) y la otra es la UI.

Siempre te va a ayudar poner algo similar para ver los tamaños.

**Reply by RodolfoMorquecho** [reply]
Gracias, las observaciones que me dieron me ayudaron mucho. Si de hecho utilizando la metáfora de una casa es como tenerla hecha y con el UI es decorar,pintar,etc. en generar darle una buena apariencia.
